apps/full-site-editing/full-site-editing-plugin/global-styles/src/dom-updater.js

```diff
@@ -25,7 +21,22 @@ export default ( options, getOptionValue ) => {
 			cssVariables[ option ] = `--${ option.replace( '_', '-' ) }`;
 		} );
 
+		let styleElement = null;
 		subscribe( () => {
+			// Do nothing until the editor is ready.
```

Comment:
Perhaps we can add here some comment saying that we must check this for iframed environments? or something along those lines. I don't know about you, but, in a few weeks, I'm going to forget that this fix was for dotcom's iframe env :)

Reply:
done :)